docker/cassandra-stress/Dockerfile-src

```diff
@@ -0,0 +1,31 @@
+FROM openjdk:8 as builder
+
+ARG BRANCH
+ARG GIT_FORK
+
+ENV BRANCH=${BRANCH:-branch-5.0}
+ENV GIT_FORK=${GIT_FORK:-https://github.com/scylladb/scylla-tools-java}
+
+RUN git clone -b ${BRANCH} --single-branch ${GIT_FORK}
+RUN apt -y update && apt-get -y install ant
+
+# revert of cassandra driver (without shard-aware)
+COPY .gitconfig /root/
+RUN cd scylla-tools-java && git revert 1deceb8a605e563f01ef1821d48316cea766da31 && git revert 87a98f0a85c9db8a9c7602e66d3d2cca56903cdf
+
+# build
+RUN cd scylla-tools-java && ant
+
+
+FROM openjdk:8 as app
+
+COPY --from=builder /scylla-tools-java /scylla-tools-java
```

Comment:
General comments on containers - up to this line, you can have a nightly job that does all the above and stores it for you in a ready-made container, with all the stuff, inc. the git clone, apt update, etc. Then you can just use it. This is a pattern I've seen elsewhere in tests as well.

The idea is to reduce runtime but also dependency on availability of external repos (you'd be amazed how many times they fail) as well as to be able to reproduce tests.

Reply:
This file here is only for cases we'll want to test a specific code or branch, for cassandra-stress.
the common case would be reusing the scylladb/scylla docker image.

those dockerfiles were added, since @roydahan asked for a documented way of using older versions if needed.

I think this would be rarely gonna be used.